src/main/java/com/djrapitops/extension/GriefDefenderExtension.java

```diff
@@ -53,7 +55,10 @@ of this software and associated documentation files(the "Software"), to deal
 )
 public class GriefDefenderExtension implements DataExtension {
 
+    private final Core api;
+
     public GriefDefenderExtension() {
+        api = GriefDefender.getCore();
```

Comment:
Remember to run `mvn test` to see that tests succeed - if they fail because of this you can create another constructor for tests